docs/swagger.yaml

```
@@ -263,4 +263,23 @@ paths:
       summary: Update user with new dat
       tags:
       - user
+  /user/me:
+    get:
+      description: Fetches the currently logged-in user's data based on the JWT token
+      produces:
+      - application/json
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)